opm/autodiff/SimulatorBase.hpp

```diff
@@ -0,0 +1,195 @@
+/*
+  Copyright 2013 SINTEF ICT, Applied Mathematics.
```

Comment:
as far as I see it, these headers do not have any legal relevance anyway, but I'll add myself.